README.md

```diff
@@ -60,70 +68,70 @@ sudo sh -c 'curl -L https://github.com/redhat-developer/odo/releases/download/v0
 </details>
 
 <details>
-<summary> :package: :checkered_flag: Windows</summary>
-
-#### In order to correctly use odo you must add it to your PATH environment variable
+<summary> Windows</summary>
 
 1. Download the `odo-windows-amd64.exe.gz` file from the [GitHub releases page](https://github.com/redhat-developer/odo/releases).
-2. Extract the file
+2. Extract the file.
 3. Add the location of extracted binary to your PATH environment variable by following [this Wiki page](https://github.com/redhat-developer/odo/wiki/Setting-PATH-variable-on-Windows).
 
 </details>
 
-#### For a list of other methods such as installing the latest mastery binary, or specific OS installations, visit the [installation page](/docs/installation.md).
-
-## Purpose
+For a list of other methods such as installing the latest binary on specific OS installations, visit the [installation page](/docs/installation.md).
```

Comment:
Binary and OS installation are different. This should be "binary, or" as it previously was.